src/components/topCarousel.tsx

```
@@ -0,0 +1,95 @@
+              emblaApi.scrollTo(index);
+            }}
+          />
+        ))}
+      </div>
+      <div className="topCarouselNavButton">
+        <button
+          onClick={() => {
+            if (!emblaApi) return;
+            emblaApi.scrollPrev();
+          }}
+        >
```

Biome (2.4.6)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)